### PR TITLE
	New components

### PR DESCRIPTION
**Pull request by Boskiantonio**
New components:
- accordion
- alert
- rating
- tabs
- pagination

### CODE REVIEW

**Comment by KrzysztofBogdan** [comment]
Please move components related with bootstrap to "bootstrap" folder

**Comment by KrzysztofBogdan** [comment]
Please merge to latest master version, there are conflicts

File: src/accordion/accordion-group.component.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    public opened: boolean = false;
+    @Input()
+    public disabled: boolean = false;
+    @Input()
+    public name: string = "group-" + nextId++;
+    @Input()
+    public header: string;
+
+    constructor(@Inject(forwardRef(() => Accordion)) public parent: Accordion) {
+        console.log(this.name);
```
Comment (KrzysztofBogdan) [inline src/accordion/accordion-group.component.ts]: Remove this

File: src/accordion/accordion-group.component.ts [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    ]
+})
+export class AccordionGroup {
+
+    @Input()
+    public opened: boolean = false;
+    @Input()
+    public disabled: boolean = false;
+    @Input()
+    public name: string = "group-" + nextId++;
```
Comment (KrzysztofBogdan) [inline src/accordion/accordion-group.component.ts]: Why this exists?
Group can not be removed by reference?

**Comment by KrzysztofBogdan** [comment]
Please replace angular imports with wildcard:
example import {ContentChildren} from "@angular/core";  ->
import * as ng from "@angular/core";

File: src/accordion/accordion-group.component.ts (hunk 1 above)
Comment (Boskiantonio) [inline src/accordion/accordion-group.component.ts]: This got removed before marge.

**Comment by Boskiantonio** [comment]
- Components moved
- Marged with latest master version
- As we discussed i changed all angular imports to be explicit.

**Comment by KrzysztofBogdan** [comment]
TsLint is not passing, 
`src/app.component.ts[77, 5]: Declaration of public instance member variable not allowed to appear after declaration of public instance member function
src/app.component.ts[96, 5]: Declaration of public instance member variable not allowed to appear after declaration of public instance member function
src/app.component.ts[104, 5]: Declaration of public instance member variable not allowed to appear after declaration of public instance member function
src/app.component.ts[126, 5]: Declaration of public instance member variable not allowed to appear after declaration of public instance member function`
Please consider dividing app.component to smaller components or turning off this inspection

File: src/bootstrap/pagination/pagination.component.ts [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    private pages: Array<number>;
+    private context: any = {};
+
+    ngOnChanges (changes: SimpleChanges): any {
+        if (changes.hasOwnProperty("data")) {
+            this.pages = Array(Math.ceil(this.data.length / this.pageSize));
+        }
+    }
+
+    private prepareContext(): any {
```
Comment (KrzysztofBogdan) [inline src/bootstrap/pagination/pagination.component.ts]: Can we move calculating context to places where currentPage and pageSize is changed? then context will be exposed as field not method in template:
[ngOutletContext]="context"

File: src/bootstrap/pagination/pagination.component.ts [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+@Component({
+    selector: "pagination",
+    templateUrl: "src/bootstrap/pagination/pagination.component.html",
+})
+export class Pagination implements OnChanges {
+
+    @Input()
+    public data: Array<any>;
+    @Input()
+    public v: string = "var";
```
Comment (KrzysztofBogdan) [inline src/bootstrap/pagination/pagination.component.ts]: Can you name this field more meaningfully?

File: src/bootstrap/rating/rating.component.ts [hunk 5]
```diff
@@ -0,0 +1,55 @@
+import {Component, EventEmitter, Output, Input, OnInit} from "@angular/core";
+
+@Component({
+    selector: "rating",
+    templateUrl: "src/bootstrap/rating/rating.component.html"})
+export class Rating implements OnInit {
```
Comment (KrzysztofBogdan) [inline src/bootstrap/rating/rating.component.ts]: Methods used in template should be public

**Pull request merged**